--- origsrc/graphite2-1.1.3/tests/vm/CMakeLists.txt	2012-05-30 05:17:49.000000000 -0500
+++ src/graphite2-1.1.3/tests/vm/CMakeLists.txt	2012-09-10 18:16:12.889417400 -0500
@@ -18,7 +18,7 @@ add_library(vm-test-common STATIC
     ${S}/Pass.cpp
     ${S}/Segment.cpp
     ${S}/Silf.cpp)
-target_link_libraries(vm-test-common graphite2 graphite2-base graphite2-segcache)
+target_link_libraries(vm-test-common graphite2-base graphite2-segcache graphite2)
 
 if  (${CMAKE_COMPILER_IS_GNUCXX})
 	set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -fomit-frame-pointer")
